HousingFinanceInterimApi/V1/Factories/UpdateTAFactory.cs

```diff
@@ -12,8 +13,8 @@ public static UpdateTADomain ToDomain(this UpdateTARequest request)
             if (request == null) return null;
             bool isTerminated;
             bool isPresent;
-            var convertToString = request.TenureEndDate.ToString();
-            if (request.TenureEndDate == null || request.TenureEndDate > DateTime.UtcNow || convertToString == "01/01/1900 00:00:00")
+            var defaultDate = new DateTime(1900, 01, 01, 0, 0, 0);
+            if (request.TenureEndDate == null || request.TenureEndDate > DateTime.UtcNow || request.TenureEndDate == defaultDate)
```

Comment:
I'd suggest `request.TenureEndDate.Date == defaultDate.Date` as whether the time is (0,0,0) shouldn't affect this